src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -490,6 +494,10 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Leaving in both the big and small version of the logo until we decide with the ux team which size we want to use.

Reply:
I think we should leave just the one we're using. Perhaps name it SonarLintLogo and we can replace it